providers/providers.tsx

```diff
@@ -23,7 +23,10 @@ export const Providers = ({ children }: PropsWithChildren) => {
           <CustomConnectKitProvider>
             <ModalProvider>
               <TxProvider>
-                <TooltipProvider delayDuration={300}>{children}</TooltipProvider>
+                <TooltipProvider delayDuration={300}>
+                  {children}
+                  <Toaster position="top-right" toastOptions={{ duration: 7000 }} />
+                </TooltipProvider>
```

Comment:
Should we move this provider above, next to modal, seems that they are quite similar, would like them to be together.

Reply:
It's kind of the same but did it anyways, let me know how it looks.